### PR TITLE
add redis persistence

### PR DESCRIPTION
**Pull request by javajax**
This is to add redis persistence to aasm.

### CODE REVIEW

**Comment by thedrow** [comment]
Any chance to get this merged?

File: lib/aasm/persistence/redis_persistence.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        base.send(:include, AASM::Persistence::RedisPersistence::InstanceMethods)
+      end
+
+      module InstanceMethods
+        # Add the inital value to intiializer
+        #
+        # redis-objects removed the key from redis when set to nil
+        def initialize(*args)
+          super
+          #require "pry"; binding.pry
```
Comment (mtylty) [inline lib/aasm/persistence/redis_persistence.rb]: This should be removed :smile:

Comment (javajax) [inline lib/aasm/persistence/redis_persistence.rb]: ops :)

**Comment by alto** [comment]
@javajax @thedrow Why the hell didn't I merge this PR? I'm sorry for this, and I simply don't know. Maybe I wanted to take a closer look and it all went out of hand... Sorry, again.

I just took a closer look and everything looks fine for me. I manually merged the PR and thereby resolved a couple of merge conflicts (which were introduced in the time this PR waited to be merged).

**Comment by alto** [comment]
I just released version `4.10.0` including your changes.

**Comment by javajax** [comment]
thanks man.
I will see about updating your projects :)

**Pull request merged**